source/get-started/install-configfile.rst

```diff
@@ -15,67 +16,61 @@ machines.
 Prerequisites
 *************
 
-Ensure that your target system supports the installation:
+For installation onto bare metal, ensure that your target system 
+supports the installation:
 
 * :ref:`system-requirements`
 * :ref:`compatibility-check`
+* :ref:`bootable-usb` of the Live Server image
 
-Process
-*******
-
-This guide describes two methods for using a configuration file with the
-clr-installer tool. You can use either method to achieve the same goal. Choose
-the method that works best for your setup.
+Example 1: Fresh installation onto bare metal
+*********************************************
 
-If you are installing |CL| for the first time, we recommend Example 1.
+This example uses a YAML configuration file to perform a new installation.
 
-To clone an existing |CL| setup on another system, we recommend Example 2.
+#. Boot up the |CL| Live Server USB thumb drive.
 
-Example 1
-=========
+#. Select :guilabel:`Clear Linux OS` from the menu.
 
-This method uses a configuration file template to perform a new installation.
+#. In the console window, log in as `root` and set a password.
 
-Perform the following steps:
+#. Verify you have a network connection to the Internet and configure proxy
+   settings if you're working behind a firewall.
 
-#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.
+#. Download a sample YAML configuration file. For example, if you want to 
+   install |CL| with a desktop GUI, you might want to use `live-desktop.yaml`_.
+   Or you can use the `live-server.yaml`_ if you want to install a non-GUI version

```

Comment:
These links didn't work when I tested it. I recommend removing them. You already show the curl commands below for both the live-server and the live-desktop.  Those commands work fine.

Reply:
fixed.